egeria-connector-jdbc/build.gradle

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+// Artifact names are taken from the directory by default, set in settings.gradle to override
+// The 'name' for the maven artifact, and description are set here
+ext.name = 'JDBC Connector'
+description = 'JDBC Connector for Egeria'
+
+dependencies {
```

Comment:
I'm thinking we need some refactoring to import the main Egeria pom (to get consistent versions) - but I can help with that in a later issue/pr :-)